project_key/models/project_project.py

```diff
@@ -130,6 +130,12 @@ def _prepare_sequence_data(self, init=True):
     def get_next_task_key(self):
         return self.sudo().task_key_sequence_id.next_by_id()
 
+    def get_task_key_sequence(self):
+        return self.sudo().task_key_sequence_id.number_next_actual
+
+    def set_task_key_sequence(self, next):
```

Comment:
Do we need these oneliners?

Reply:
@alexey-pelykh Actually, yes we do because this oneliner method enables you to override just that part of the code/logic in your extension module in case you want to have different functionality. For example, functionality you've mentioned above where you'd like to have task keys without "blanks" or implement some completely different logic.

Reply:
`set_next`/`get_next`?